frontends/web/src/components/headerssync/headerssync.tsx

```diff
@@ -80,3 +92,10 @@ export default class HeadersSync extends Component {
         );
     }
 }
+
+const subscribeHOC = subscribe<SubscribedHeadersSyncProps, HeadersSyncProps & TranslateProps>(({ coinCode }) => ({
+    status: `coins/${coinCode}/headers/status`,
+}), false, true)(HeadersSync);
```

Comment:
why `..., false, true`? Before it seemed to be the opposite.

Reply:
will open a separate PR after this is merged

Reply:
why?